### ISSUE TITLE
New command: Create a new untitled Excalidraw file

### ISSUE THREAD

**Issue by pomdtr** [excalidraw/excalidraw-vscode#21]
We should also integrate with the `Create: New File...` command.

**Comment by rebornix** [comment]
Hi @pomdtr , VS Code dev here. I can help with this if you haven't started yet.

**Comment by pomdtr** [comment]
Hi @rebornix ! Thanks for proposing your help, go ahead :)

**Comment by pomdtr** [comment]
I did take a stab at it some time ago, but I did encouter a bug when saving the untitled document : the untitled editor was not closing after saving it to a document, leading to two document opened :/

**Comment by aadimator** [comment]
Waiting for this eagerly. You should also specify a setting to set the "default path" where it should store the newly created files.

**Comment by Acumane** [comment]
Emphasis on unsaved/untitled. My ideal workflow is embedding drawings into markdown, so I've no need for saving to its own file

**Comment by rajguru7** [comment]
Hi,
I am not sure if it is related to this issue, but I am not able to create a `*.excalidraw.svg`, `*.excalidraw.png`, `*.excalidraw` file using the below command in terminal:
```sh
code <file-you-want-to-create>.excalidraw
```
Please let me know if it is possible.

A workaround for this is to open code in the same directory using `code .` and then creating a new file inside vscode with the `.excalidraw` extension.

**Comment by pomdtr** [comment]
Yeah there is something wrong in the way the extension handle files. I really want to fix it, but I don't have a lot of time to dedicate to the issue right now. 

Any help would be appreciated! #85 is probably related to this.

**Comment by sammcj** [comment]
Came here to log a feature request for exactly this.

It's a bit of a pain having to create a new file with a specific extension, open it with vscode etc... and it means there's no hotkey assignable shortcut that can be used with automations/layouts etc....

**Comment by the-yojimbo** [comment]
anyone have any update on this request? currently, I have to make a copy an existing .excalidraw file and then I can open it from Excalidraw. I wish I could actually help but I am no programmer.
